File: 01_introductions.js
// Converter from Fahrenheit to Celsius
const values_fahrenheit = [65, 80, 92, 70, 63, 72];
const values_celsius = [];
for (const element of fahrenheit) {
  let value_celsius = (element - 32) * (5/9);
  values_celsius.push(value_celsius);
}
document.write("Values in Fahrenheit: " + values_fahrenheit + "<br>")
document.write("Values in Celsius: " + values_celsius + "<br>")

File: 01_introductions.py
#!/usr/bin/env python3

# Python 3.9.5

# 01_introductions.py

values_fahrenheit = [65, 80, 92, 70, 63, 72]
values_celsius = [round((value - 32) * (5/9), 2) for value in values_fahrenheit]

print("Values in Fahrenheit: " + str(values_fahrenheit))
print("Values in Celsius: " + str(values_celsius))
